src/components/markdown-renderer/replace-components/markmap/markmap-frame.tsx

```diff
@@ -12,14 +12,24 @@ export const MarkmapFrame: React.FC<MarkmapFrameProps> = ({ code }) => {
       return
     }
     const actualContainer = diagramContainer.current
-    Promise.all([import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/transform'), import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/view')])
-      .then(([transform, view]) => {
+    Promise.all([import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/transform'), import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/view'), import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/util/loader')])
+      .then(([transform, view, loader]) => {
         const svg: SVGSVGElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
         svg.setAttribute('width', '100%')
         actualContainer.querySelectorAll('svg').forEach(child => child.remove())
         actualContainer.appendChild(svg)
-        const data = transform.transform(code)
-        view.Markmap.create(svg, {}, data)
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const { root, features } = transform.transform(code)
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
```

Comment:
Why do you have these ESLint rules disabled? "No unsafe assignment" sounds as you didn't install/include the proper types.

Reply:
Well there are official types, but they use any

Reply:
![cry](https://media1.giphy.com/media/1BXa2alBjrCXC/giphy.gif)

Reply:
So leave it as is?

Reply:
Well. The alternative would be to add additional types. If you want to do that.. feel free.

Reply:
So I guess you don't want to do that? :sweat_smile: 
It's okay for me, but I need to know if I should wait.

Reply:
I'm currently working on that, but I will do so in their repo and we fix this with their next update, I hope